include/sentry.h

```diff
@@ -1298,6 +1298,30 @@ SENTRY_API void sentry_options_set_database_pathw(
     sentry_options_t *opts, const wchar_t *path);
 SENTRY_API void sentry_options_set_database_pathw_n(
     sentry_options_t *opts, const wchar_t *path, size_t path_len);
+
+/**
+ * Allows users to define a thread stack guarantee manually on each thread. This
+ * is necessary for crash handlers to run after a thread crashed due to a
+ * stack-overflow on Windows.
+ *
+ * By default the Native SDK automatically sets this value for you, but in some
+ * cases it might be preferable to set these values yourself for each thread you
+ * manage. Ensure to disable the `SENTRY_THREAD_STACK_GUARANTEE_AUTO_INIT` CMake
```

Comment:
```suggestion
 * manage. Make sure to disable the `SENTRY_THREAD_STACK_GUARANTEE_AUTO_INIT` CMake
```

Reply:
Can I ask why? I consider "ensure" to be more formal and shorter, both great attributes for documentation.

Reply:
For some reason "ensure to" sounds a bit weird, but I cannot find grammatical backings for this suggestion so I'm fine with keeping it as-is.